changelog/7089.bugfix.md

```diff
@@ -0,0 +1,6 @@
+Fix `ConveRTTokenizer` failing because of wrong model URL by making `model_url` parameter of `ConveRTTokenizer` mandatory.
```

Comment:
🤔 
```suggestion
Fix  `ConveRTTokenizer` failing because of wrong model URL by making the `model_url` parameter of `ConveRTTokenizer` mandatory.
```